workers/unity/Assets/Playground/Scripts/Worker/AndroidClientWorkerConnector.cs

```diff
@@ -0,0 +1,60 @@
+using Improbable.Gdk.Core;
+using Improbable.Gdk.Mobile;
+#if UNITY_ANDROID
+using Improbable.Gdk.Mobile.Android;
+#endif
+using System;
+using UnityEngine;
+
+
+namespace Playground
+{
+    public class AndroidClientWorkerConnector : MobileWorkerConnector, IMobileConnectionController
+    {
+        public string IpAddress { get; set; }
+        public ConnectionScreenController ConnectionScreenController { get; set; }
+
+        [SerializeField] private GameObject level;
+
+        private GameObject levelInstance;
```

Comment:
Is the platform specific worker connectors meant to be just for the playground workers or used more generally? If used more generally then non platform stuff like this can ve moved into the mobile worker connector. However then I'd possibly recommend having platform specific worker connectors (that just do the platform specific stuff) then having the specific playground version doing stuff like the level instances, as that is a specific opinion that the main connector has gone away from.

Reply:
not sure I'm understanding this

Reply:
These platform specific worker connectors are meant to be just for the playground workers.